hpolib/abstract_benchmark.py

```diff
@@ -140,6 +140,20 @@ def wrapper(self, configuration, **kwargs):
             return foo(self, config_array, **kwargs)
         return wrapper
 
+    @staticmethod
+    def _configuration_as_dict(foo):
```

Comment:
Do we use this anywhere?

Reply:
Also something that came to my mind: If we apply "_check_configuration", we do not change the type of the argument, e.g. transforming an array to a dictionary, right? Shouldn't we do this?

Reply:
Yes, we use this function in the learna benchmark.  

I had an error by passing a configuration to an objective function. So I thought it might be interesting to write a short wrapper which converts a configspace configuration to a dictionary automatically. So one does not to think about it anymore.

Reply:
Great. I also openend issue #10 such that we don't forget to change this for all benchmarks. If there is anything to be changed, please do so in a separate PR